geoai/universat.py

```
@@ -0,0 +1,244 @@
+
+# Auto-clone UniverSat repo and setup environment
+UNIVERSAT_CACHE_DIR = os.path.expanduser("~/.cache/geoai/UniverSat")
+_src = os.path.join(UNIVERSAT_CACHE_DIR, "src")
+if not os.path.exists(UNIVERSAT_CACHE_DIR):
+    subprocess.run(
+        [
+            "git",
+            "clone",
+            "https://github.com/gastruc/UniverSat.git",
+            UNIVERSAT_CACHE_DIR,
+        ],
```

Ruff (0.15.21)

[error] Starting a process with a partial executable path

(S607)